### PR TITLE
add klaytn mainnet cypress and testnet baobab

### PR DESCRIPTION
**Pull request by toniya-klaytn**
https://chainlist.org/chain/1001
https://chainlist.org/chain/8217

### CODE REVIEW

**Comment by toniya-klaytn** [comment]
recheck

**Comment by toniya-klaytn** [comment]
- - -
   ***I have read the CLA Document and I hereby sign the CLA***
- - -

**Comment by toniya-klaytn** [comment]
recheck

File: gnosis/eth/clients/blockscout_client.py [hunk 1]
```diff
@@ -42,6 +42,8 @@ class BlockscoutClient:
         EthereumNetwork.GODWOKEN: "https://v1.gwscan.com/",
         EthereumNetwork.VENIDIUM_TESTNET: "https://evm-testnet.venidiumexplorer.com/",
         EthereumNetwork.VENIDIUM: "https://evm.venidiumexplorer.com/",
+        EthereumNetwork.KLAYTN_TESTNET: "https://baobab.scope.klaytn.com/",
+        EthereumNetwork.KLAYTN_MAINNET: "https://scope.klaytn.com/",
```
Comment (Uxio0) [inline gnosis/eth/clients/blockscout_client.py:46]: Is this blockexplorer blockscout? Will API work in the same way?

Comment (toniya-klaytn) [reply gnosis/eth/clients/blockscout_client.py:46]: Yes we have we have a testnet and mainnet. Please check https://chainlist.org/chain/8217 and https://chainlist.org/chain/1001. 
We have also deployed safe contracts and merged. https://github.com/safe-global/safe-deployments/pull/95

More References: 
**Mainnet**
Network Name: Klaytn Cypress
RPC URL: (Default: https://public-node-api.klaytnapi.com/v1/cypress)
Block Explorer URL: https://scope.klaytn.com/
Chain ID: 8217

**Testnet**
Network Name: Klaytn Baobab
RPC URL: https://api.baobab.klaytn.net:8651/
Block Explorer URL: https://baobab.scope.klaytn.com/
Chain ID: 1001 

**Documentation**
https://docs.klaytn.foundation/

**Comment by Uxio0** [comment]
CLA is still not passing, you have to paste "I have read the CLA Document and I hereby sign the CLA", just that, no placeholders or text formatting

**Comment by toniya-klaytn** [comment]
I have read the CLA Document and I hereby sign the CLA"

**Comment by toniya-klaytn** [comment]
recheck

**Comment by Uxio0** [comment]
Networks need to be added to https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py

**Comment by toniya-klaytn** [comment]
Network is already added : 
https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L111
https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L146

**Comment by Uxio0** [comment]
> 

Then you should use the same names

**Comment by toniya-klaytn** [comment]
Thanks for the feedback. Renamed as requested

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**